capnpy/segment/_copy_pointer.h

```diff
@@ -2,10 +2,16 @@
 #define RAISE_OUT_OF_BOUNDS __pyx_f_6capnpy_7segment_7builder_raise_out_of_bounds
 static PyObject* RAISE_OUT_OF_BOUNDS(Py_ssize_t, Py_ssize_t);
 
+#if PY_MAJOR_VERSION < 3
```

Comment:
can't we reuse the content of `_util.h` instead of rewriting?
Also, maybe we could rename it into `_py_compat.h` or similar?